psi4/driver/qcdb/vib.py

```diff
@@ -58,7 +57,7 @@ def _success(label):
         """Function to print a '*label*...PASSED' line to screen.
         Used by :py:func:`util.compare_values` family when functions pass.
         """
-        msg = '\t{0:.<66}PASSED'.format(label)
+        msg = f'\t{label:.<66}PASSED'
```

Comment:
Is this going to be the python string formatting standard we're agreeing on?

Reply:
There hasn't been a formatting standard btwn `%` and `.format`, partly because the core devs were in two camps. But everyone loves f-strings, so now that we're py36, things are migrating as opportunity allows. Personally, I do f-strings unless the insides are too complicated such that I think format looks cleaner. And I'd still do a simple % if the string involved a lot of curly brackets.